src/components/modal/index.tsx

```diff
@@ -81,6 +77,28 @@ const Modal: FC<ModalProps> = ({
             "h-full": size === "fullscreen",
           })}
         >
+          {title ? (
+            <div className="flex items-center pb-15">
+              <h2 className="font-bold text-xl text-grey-dark w-12/12">
+                {title}
+              </h2>
+              <div
+                className="z-modalClose cursor-pointer"
+                onClick={handleClose}
+                data-testid="modal-close"
+              >
+                <CloseMIcon color={style === "dark" ? "#FFFFFF" : "#232A36"} />
+              </div>
+            </div>
+          ) : (
+            <div
+              className="absolute z-modalClose cursor-pointer right-modalClose top-modalClose"
+              onClick={handleClose}
+              data-testid="modal-close"
+            >
+              <CloseMIcon color={style === "dark" ? "#FFFFFF" : "#232A36"} />
+            </div>
```

Comment:
the whole DOM for the close is duplicated, can you share it between the two branches of the condition? :)